applications/areg-portal/backend/main.py

```diff
@@ -95,28 +91,90 @@ async def has_access(credentials: HTTPBasicCredentials = Depends(security)):
 # Operations
 
 
-@prefix_router.get('/live', response_model=str, tags=['test'])
-def live() -> str:
```

Comment:
Should keep liveness and readiness endpoints

Reply:
Added them to the api and regenerated everything [7ecb94f](https://github.com/MetaCell/scicrunch-antibody-registry/pull/41/commits/7ecb94f7c2256761cbc68294fcd8c8b8a1607bc7)